### PR TITLE
Add webP to allowed format on the DropZone help

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Add WEBP to allowed formats on the help text.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
I restart the tests but I think it will be fixed only on Monday

Edit : fixed

**Review by marsaldev — DISMISSED** [review]

**Review by MeKeyCool — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
I guess this is a PR that we can obviously merge without QA, wdyt @PrestaShop/qa-team ?

Of course, after the wording gets validated :)

**Comment by HanaRebaiQA** [comment]
Hello @NeOMakinG 

We, @PrestaShop/qa-team, think that we should do a quick check for this PR before being merged.

Thank you!

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.dropImages' => $this->trans('Drop images here', 'Admin.Catalog.Feature'),
                 'window.selectFiles' => $this->trans('or select files', 'Admin.Catalog.Feature'),
                 'window.recommendedSize' => $this->trans('Recommended size 800 x 800px for default theme.', 'Admin.Catalog.Feature'),
-                'window.recommendedFormats' => $this->trans('JPG, GIF or PNG format.', 'Admin.Catalog.Feature'),
+                'window.recommendedFormats' => $this->trans('JPG, GIF, PNG or WEBP format.', 'Admin.Catalog.Feature'),
```
Comment (kpodemski) [inline src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php]:
```suggestion
                'window.recommendedFormats' => $this->trans('JPG, GIF, PNG or WebP format.', 'Admin.Catalog.Feature'),
```

File: src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig [hunk 2]
```diff
@@ -321,7 +321,7 @@
 "Drop images here": "Drop images here"|trans({}, 'Admin.Catalog.Feature'),
 "or select files": "or select files"|trans({}, 'Admin.Catalog.Feature'),
 "files recommandations": "Recommended size 800 x 800px for default theme."|trans({}, 'Admin.Catalog.Feature'),
-"files recommandations2": "JPG, GIF or PNG format."|trans({}, 'Admin.Catalog.Feature'),
+"files recommandations2": "JPG, GIF, PNG or WEBP format."|trans({}, 'Admin.Catalog.Feature'),
```
Comment (kpodemski) [inline src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig]:
```suggestion
"files recommandations2": "JPG, GIF, PNG or WebP format."|trans({}, 'Admin.Catalog.Feature'),
```

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php (hunk 1 above)
Comment (kpodemski) [reply src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php]: Maybe it's a nitpick, but eventually, it is WebP, not WEBP. What do you think @PrestaEdit ?

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thx @PrestaEdit - aaand, there's no need for wording check in this one :)

**Review by atomiix — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-team can we merge it? it's just wording modification

**Comment by Robin-Fischer-PS** [comment]
Direct merge or check by a dev before, your choice (ping @NeOMakinG :D )

**Comment by kpodemski** [comment]
Thanks @PrestaEdit

**Pull request merged**